My Cloud EMS/employer.c

```diff
@@ -0,0 +1,451 @@
+        {
+            passDB(stud.sysID, pass, choice); // Passed Student's System ID
+        }
+    }
+}
+void passStrength(char pass[25]) // Checks the strength of Password (Need some modifications)
+{
+    int i = 0, check = 0, spclSymb = 0, num = 0, upCase = 0, lowCase = 0;
+    char x;
+    if (strlen(pass) <= 8)
+        printf("Password must be atleast 8 Characters\n");
+    else if (strlen(pass) > 20)
+        printf("Password must be atmost 20 Characters\n");
+    else
+    {
+        while (pass[i] != '\0')
+        {
+            x = (int)pass[i];
+            if (x >= 97 && x <= 122)
+                lowCase = 1;
+
+            if (x >= 65 && x <= 90)
+                upCase = 1;
+
+            if (x >= 48 && x <= 57)
+                num = 1;
+
+            if ((x >= 32 && x <= 47) || (x >= 58 && x <= 64) || (x >= 91 && x <= 96) || (x >= 123 && x <= 126))
+                spclSymb = 1;
+
+            i++;
+        }
+
+        if (lowCase)
+            check += 10;
+        if (upCase)
+            check += 10;
+        if (num)
+            check += 10;
+        if (spclSymb)
+            check += 10;
+
+        if (check == 40)
+            printf("Password Strength: Excellent \n");
+        else if (check == 30)
+            printf("Password Strength: Very Good\n");
+        else if (check == 20)
+            printf("Password Strength: Average\n");
+        else
+            printf("Password Strength: Poor\n");
+    }
+}
+
+void passDB(unsigned long long sysID, char pass[], int choice) // Store ID Password into respective database
+{
+    if (strcmp(getUserType(choice), "Employee") == 0)
+    {
+        fPtr = fopen("EmpPassDataBase.txt", "a"); // Opening Employee's Database
+    }
+    else if (strcmp(getUserType(choice), "Student") == 0)
+    {
+        fPtr = fopen("StPassDataBase.txt", "a"); // Opening Stident's Database
+    }
+    else
+    {
+        printf("ERROR... Can't find database:(");
+    }
+    if (fPtr == NULL)
+    {
+        printf("\nCan't Open File ");
+        exit(1);
+    }
+    else
+    {
+        fflush(stdin);
+        fprintf(fPtr, "\n%llu,%s", sysID, pass); // Writing id pass into text file
+        fclose(fPtr);                            // Closing text file
+        printf("\nPassword has been Set.");
+    }
+    fPtr = NULL; // Resetting FILE pointer to NULL
+}
+
+void accInfoDB(unsigned long long sysID, char name[41], char dgreName[15], int batch, char brnchName[40], char spclName[50], char section[6], char group[4])
+{
+    fPtr = fopen("StudAccDtlsDB.txt", "a");
+    if (fPtr == NULL)
+    {
+        printf("\nCan't Open File ");
+        exit(1);
+    }
+    else
+    {
+        fflush(stdin);
+        int i;
+
+        fprintf(fPtr, "\n\nName: %s\nSystem Id: %llu\n\n\tDegree name: %s\n\tBatch: %d\n\tBranch name: %s\n\tSpecialization: %s\n\tSection: %s\n\tGroup: %s\n\n", name, sysID, dgreName, batch, brnchName, spclName, section, group);
```

Comment:
Use individual fprintf() statement for printing the same. It will enhance the readability of the code.